README.md

```diff
@@ -1,24 +1,13 @@
-# README
-
-This README would normally document whatever steps are necessary to get the
-application up and running.
-
-Things you may want to cover:
+# Potres-app
 
 * Ruby version
+2.7.1
 
-* System dependencies
-
-* Configuration
-
-* Database creation
-
-* Database initialization
-
-* How to run the test suite
+## How to contribute
 
-* Services (job queues, cache servers, search engines, etc.)
+If you want to contribute, please check out our issues tab, see if anything is approved and comment on that issue so we know someone is working on it.
```

Comment:
Maybe apart from commenting to assign yourself to the issue when starting working on it?